handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
+
+**Laptop (non-engineers):** $2000
+
+**Laptop coverage:** $380
+
+**Monitor:** $380
+
+**Headphones:** $200
+
+**Keyboard:** $130
+
+**Mouse or trackpad:** $80 or $130
+
+**USB-C Adapter:** $70
+
+**HDMI/monitor cable:** $15
+
+**Laptop stand:** $40
+
+**Desk:** $500
+
+**Ergonomic chair:** $200
+
+
+
+## Ongoing expenses
+**Software:** case by case, please get your manager’s approval.
+
+**Internet/WiFi plan:** $60/month
+
+**Cell phone plan:** $75/month (no need to expense, this is automatically added onto your pay).
+
+**Coworking/office space:** TBD
+
+**Professional development (conferences, courses):** $1000/year
+
+**Moving (moving, packing, shipping, etc.):** $1000/year
+
+**Wellness:** $85/month
+
+
+
+## Other expenses:
+
+**Laptop repairs:** if your laptop requires repairs, we cover up to $500 a year on repairs. If necessary repairs exceed the yearly budget please reach out to PeopleOps for approval.
```

Comment:
This doesn't imply that the teammates own the laptop, does it? I can't recall our current policy here.

Reply:
I don’t understand why there should be a cap on repair costs for our work laptop?

Reply:
@dadlerj Correct, it would still be Sourcegraph property. @ggilmore I should clarify, what I meant here was that it would be $500/year for repairs without needing approval, but anything above that would require additional approval before expensing. Will edit that to be more clear :)